cmd/monaco/generate/deletefile/deletefile.go

```diff
@@ -257,11 +261,42 @@ func createConfigAPIEntry(c config.Config, apis api.APIs, project project.Projec
 			return persistence.DeleteEntry{}, fmt.Errorf("resolved name parameter is no string")
 		}
 		scopeValue = nameOfRefCfgStr
+
+		if apis[c.Coordinate.Type].ID == api.KeyUserActionsWeb {
+			// while generating the delete file (which happens offline) we are not able
+			// to resolve that parameter, so we delete it
+			delete(c.Parameters, config.ScopeParameter)
+			props, resolvErr := c.ResolveParameterValues(entities.New())
+			if resolvErr != nil {
+				return persistence.DeleteEntry{}, errors.Join(resolvErr...)
+			}
+			rendered, err := c.Render(props)
+			if err != nil {
+				return persistence.DeleteEntry{}, err
+			}
+
+			var jsonM map[string]string
+			err = json.Unmarshal([]byte(rendered), &jsonM)
+			if err != nil {
+				return persistence.DeleteEntry{}, err
+			}
+
+			customValues["domain"] = jsonM["domain"]
+			customValues["actionType"] = jsonM["actionType"]
+		}
 	}
 
+	for _, s := range []string{"project", "type", "id", "name", "scope"} {
+		if _, ok = customValues[s]; ok {
```

Comment:
I don't get the point of this loop and the condition seems to not be covered by tests. 
You only ever write `domain` and `actionType` into the customValues map, but then check that it does not contain `project`, `type`, etc. keys - which I won't unless I'm missing where you've written them into the map. Did you mean to validate the content of the fields?

Reply:
If you reuse the customValues for a later use case (for which CustomValues was built) you are not allowed to use theses fields, as it would cause a crash of the program.

Reply:
maybe it would be better to error out here tho 🤔